addons/binding/org.openhab.binding.sensebox/ESH-INF/thing/channels.xml

```diff
@@ -157,11 +157,11 @@
 		"sensorType": "SDS 011",
 	-->
 	<channel-type id="particulateMatter2dot5">
-		<item-type>Number</item-type>
+		<item-type>Number:Density</item-type>
 		<label>Particulate Matter - PM2.5</label>
-		<description>Current amount of particles less than 2.5 µm in diameter</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f µg/m³">
+		<description>Current density of particles less than 2.5 µm in diameter</description>
+		<category>particulateMatter</category>
```

Comment:
The category of channels should not be specified if it doesn't match any of the ESH [Channel Categories](https://www.eclipse.org/smarthome/documentation/concepts/categories.html#channel-categories).

Reply:
Thank you for the pointer, invalid categories removed.